data/json/items/tools.json

```diff
@@ -405,10 +405,10 @@
     "name": "fire extinguisher",
     "description": "This is a large emergency fire extinguisher.  It would be useful for putting out adjacent fires.",
     "price": 5000,
-    "material": "aluminum",
-    "weight": 9100,
+    "material": "steel",
```

Comment:
Changing to steel and (much) lower weight?
It's supposed to be a "large emergency fire extinguisher".